aztk_cli/utils.py

```diff
@@ -50,6 +50,7 @@ def print_cluster(client, cluster: models.Cluster, internal: bool = False):
     log.info("------------------------------------------")
     log.info("State:          %s", cluster.state.value)
     log.info("Node Size:      %s", cluster.vm_size)
+    log.info("Created:        %s", cluster.pool.creation_time.strftime('%Y-%m-%d %H:%M:%S'))
```

Comment:
can you make an helper function for converting the date to string so we only provide the format once?

Reply:
Sure. I didn't include the seconds in the table listing to keep the width down, so added a parameter for that, but can remove it if you'd prefer to keep it simple.